docs/source/using/run-against-an-existing-machine.adoc

```diff
@@ -84,9 +84,16 @@ Use the following command on the host to run {project} against a remote machine:
 
 ----
 $ minishift start --vm-driver generic --remote-ipaddress <remote_IP_address> --remote-ssh-user <username> --remote-ssh-key <private_ssh_key>
+$ minishift addon apply htpasswd-identity-provider
 ----
 
 [NOTE]
 ====
 The value of the `--remote-ssh-key` flag must be the location of a private SSH key on the host machine.
 ====
+
+[NOTE]
+====
+Make sure you apply `htpasswd-identity-provider` add-on after minishift start, it will allow you to use htpasswd as identity provider.
+By default minishift uses `Allow All` identity provider, which allows any non-empty user name and password to log in.
```

Comment:
The add-on `htpasswd-identity-provide` will allow you to use htpasswd as identity provider instead of `Allow All` identity provider.

Reply:
cc @robin-owen take a look and suggest.